.github/labeler.yml

```diff
@@ -234,6 +234,11 @@
     - "subsys/bluetooth/**/*"
     - "!subsys/bluetooth/mesh/**/*"
 
+"CI-gazell-test":
+  - "include/gz*.h"
+  - "samples/gazell/**/*"
+  - "subsys/gazell/*"
```

Comment:
Note that if this folder is refactored to contain subfolders in the future, then content within those subfolders won't be detected.